README.md

```diff
@@ -1900,6 +1915,53 @@ The 'group' owner of the tomcat war file.
 
 Default value: 'tomcat'.
 
+#### `tomcat_install`
```

Comment:
please add the source_url parameter to the README

Reply:
Actually, since the `tomcat_install` resource type will be private (as requested in https://github.com/puppetlabs/puppetlabs-tomcat/pull/260/files#r154203710) it wouldn't have any documented parameters, but instead would have a section to mention such like:
```markdown
### Private Resource types
* `tomcat_install`: Installs the software into the given directory.
```